downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -21,12 +21,12 @@ The TLSv1.3 parameter only works when OpenSSL 1.1.1 or higher is used.
 If `nginx_tls-protocols = ['TLSv1.3']` only TLSv1.3 is enabled. To set more than one protocol use `nginx_tls_protocols = ['TLSv1.2', 'TLSv.1.3']`.
 
 Default = `TLSv1.2`.
-| *`nginx_user_http_config`* | List of nginx configurations for `/etc/nginx/nginx.conf` under the http section. 
+| *`nginx_user_http_config`* | | List of nginx configurations for `/etc/nginx/nginx.conf` under the http section. 
 
 Each element in the list is provided into `http nginx config` as a separate line. 
 
 Default = empty list
-| *`registry_password`* | `registry_password` is only required if a non-bundle installer is used.
+| *`registry_password`* |`registry_password` |`registry_password` is only required if a non-bundle installer is used.
```

Comment:
Just a suggestion but I find it reads better if you include the article here

"The `registry_password` is required if a non-bundle installer is used"